components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -212,14 +214,16 @@ export class ProjectEffects {
 
   @Effect()
   getDormantApplyRulesStatus$ = observableInterval(1000 * DORMANT_RULE_STATUS_INTERVAL).pipe(
+    withLatestFrom(this.store.select(allPerms)),
     withLatestFrom(this.store.select(applyRulesStatus)),
-    filter(([_, { state }]) =>
+    filter(([[_interval, _allPerms], { state }]) =>
       state === ApplyRulesStatusState.NotRunning
     ),
-    switchMap(() => [
-      new GetProjects(),
-      new GetApplyRulesStatus()
-    ]),
+    switchMap(([[_interval, list], _status]) => {
+      return get(['/iam/v2/projects', 'get'], list) ?
+        [new GetProjects(), new GetApplyRulesStatus()] :
+        [new GetApplyRulesStatus()];
+    }),
```

Comment:
This does the periodic polling of GetProjects. This uses introspection yet a third time, and if there is permission does both GetProjects and GetApplyRulesStatus, otherwise just GetApplyRulesStatus.